.github/workflows/testing.yaml

```diff
@@ -95,10 +99,15 @@ jobs:
 
       - name: Install requirements
         run: make deps
-        if: steps.cache-rocks.outputs.cache-hit != 'true'
```

Comment:
This line was needed so as not to reinstall the rocks every time. Why was it removed?

Reply:
Because `make deps` was not run for tarantool 3.1 case.

Reply:
Oh, I got it. You could clean the cache here:
https://github.com/tarantool/metrics-export-role/actions/caches

Please, revert the change than.